Content.Client/Weapons/Ranged/Systems/GunSystem.cs

```diff
@@ -310,7 +310,7 @@ protected override void CreateEffect(EntityUid uid, MuzzleFlashEvent message, En
 
         _animPlayer.Play(ent, anim, "muzzle-flash");
         var light = EnsureComp<PointLightComponent>(uid);
-        light.NetSyncEnabled = false;
+        light.NetSyncEnabled = true;
```

Comment:
Might be better to just only disable netsyncing if the pointlight doesn't exist, I'll play around with it.